src/index.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import SignUp from './signup';
+import SignIn from './SignIn';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  useParams,
+} from 'react-router-dom';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
+      <Switch>
+        <Route exact path="/vote" component={App} />
```

Comment:
```suggestion
        <Route exact path="/vote"><App /></Route>
```
Route 내부에 렌더링해야 하는 컴포넌트가 많아지면 prop으로 전달하는 것 보다 이렇게 하는 방법도 있어 참고해두면 좋을 것 같습니다